ClientHints.php

```diff
@@ -188,10 +188,12 @@ public function getOperatingSystemVersion(): string
     public function getBrandList(): array
     {
         if (\count($this->fullVersionList)) {
```

Comment:
add the is_array check before using count

Reply:
will do that.

Reply:
maybe `count() > 0` is superfluous, I don't know exactly what will happen if 2 arrays are empty in `array_combine`